profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.install

```diff
@@ -52,3 +53,42 @@ function multisite_drupal_toolbox_update_7201() {
   multisite_config_service('filter')->enableTextFilter('filtered_html', 'toolbox_sanitize');
   multisite_config_service('filter')->enableTextFilter('full_html', 'toolbox_sanitize');
 }
+
+/**
+ * NEPT-1453: Enable HTML sanitize filter for all "non-checkplain" formats.
+ */
+function multisite_drupal_toolbox_update_7202() {
+  $formats = filter_formats();
+  $format_machine_names = array_keys($formats);
+  foreach ($format_machine_names as $format_machine_name) {
```

Comment:
I do not receive the same kind of object and the source that give the input of the process is not the same.
As the hook_updatedb is a one shot, I did not spend much more to centralize the process because the investment is high compare to the benefice.

Reply:
ok